build.psm1

```diff
@@ -1012,7 +1012,7 @@ function Start-PSPester {
     # All concatenated commands/arguments are suffixed with the delimiter (space)
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-linux`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Private.Xml.Linq.dll; SideIndicator=<=}, @{InputObject=System.Threading.Timer.dll; SideIndicator=<=}, @{InputObject=System.Xml.XDocument.dll; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 123
123:         $diffs | Should -BeExactly $null
```